java/src/main/java/com/powsybl/dataframe/network/adders/NetworkElementAdder.java

```
@@ -27,5 +28,15 @@ public interface NetworkElementAdder {
      * The first dataframe is considered the "primary" dataframe, other dataframes
      * can provide additional data (think steps for the tap changers).
      */
-    void addElements(Network network, List<UpdatingDataframe> dataframes);
+    default void addElements(Network network, List<UpdatingDataframe> dataframes) {
+    }
+
+    default void addElementsWithBay(Network network, List<UpdatingDataframe> dataframe, boolean throwException, Reporter reporter) {
+    }
```

Would be better to throw an unsupported operation exception by default